packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -210,6 +211,20 @@ function isSideloadingSupportedForWebHost(app: OfficeApp): boolean {
   return false;
 }
 
+async function getOutlookVersion(): Promise<string | undefined> {
+  try {
+    const OutlookInstallPathVersionRegistryKey = `HKEY_LOCAL_MACHINE\\SOFTWARE\\Microsoft\\Office\\ClickToRun\\Configuration`;
+    const key = new registry.RegistryKey(`${OutlookInstallPathVersionRegistryKey}`);
+    const outlookInstallVersion: string | undefined = await registry.getStringValue(key, "ClientVersionToReport");
+
+    return outlookInstallVersion;
+  }
+  catch (err) {
+    const errorMessage: string = `Unable to find Outlook version location: \n${err}`;
```

Comment:
We need to be careful here. There are other ways to install Outlook, and we don't want the inability to get the version number to block usage there. I would recommend this function not throw an error but just return undefined if it cannot get the version.